pkg/virt-controller/services/template.go

```diff
@@ -144,6 +149,15 @@ func isSRIOVVmi(vmi *v1.VirtualMachineInstance) bool {
 	return false
 }
 
+func IsCPUNodeDiscoveryEnabled(store cache.Store) bool {
+	var value string
+	value, _ = getConfigMapEntry(store, featuregates.FEATURE_GATES_KEY)
```

Comment:
Good question what to do when an error occurs here, but we should handle it explicitly in here.